src/interfaces/utils-interfaces/address-schema-interface.ts

```diff
@@ -0,0 +1,9 @@
+export interface AddressSchemaIF {
+  streetAddress: {}
```

Comment:
Why use `{}` instead of `any` (or strings) ?

Reply:
Oh, these are rules?

Reply:
Oh, no, this is an interface. Hmm.

Reply:
OK, ok, I get it... each item is an object of validation rules. I dunno how best to type this.

Reply:
Me either. Didn't want to over-think it. 
At the very least i wanted to type the base properties but internally these objects can contain rules.. functions... properties.. decorators.. so i dunno. 
Just enforcing the base schema properties i think is ok for now.